docs/quick-start.md

```diff
@@ -3,10 +3,226 @@ title: Quickly starting with eScriptorium
+<!-- example images are taken from https://gallica.bnf.fr/iiif/ark:/12148/bpt6k10750420/manifest.json -->
+
+Let's import images [from the local file system](import.md#1-from-the-local-file-system) in order to process them with eScriptorium.
+
+First, you need to download the file located [here](files/demo_images.zip) and unzip the resulting file.
+
+It contains 5 images taken from the French national Library's online library, [Gallica](https://gallica.bnf.fr), in particular from the document called "[Gallery of Fashion](https://gallica.bnf.fr/ark:/12148/bpt6k10750420)", printed in London in january 1800. The images contain text blocks and illustrations with legends.
+
+![image: screenshot of the file explorer showing the 5 image files in the downloadable archive](img/quick_start/demo_images_content.png "There are five images in the downloadable demo file, all taken from the same Gallica document.")
+
+To import them in the application, got to the corresponding document's dashboard and click on the "Images" tab (`{base_url}/document/{document_id}/images/`). Click in the "dropping" area to open the file explorer and select the images. Do not refresh the page until all the images are loaded! 
+
+For each of the image, a new thumbnail appears. If you go back to the project's dashboard, you can see that our document now contains 5 images.
+
+![image: animation showing the process to open a document and import images, following the instruction given above. At the end, we show that the document now contains 5 images](img/quick_start/import_images.gif "Once import is over, we should have 5 images in the document we previously created")
+
+## Detecting the Layout of the documents
+
+Now that the images are loaded on the application, we need to apply a segmentation model in order to detect the layout of the documents and locate the lines of text. This step is necessary before applying a recognition model to predict the transcription.
+
+Let's go back to the document's dashboard and click on the "Images" tab (`{base_url}/document/{document_id}/images/`). Each thumbnail allows you to interact with one of the document's pages. Select the first page by clicking on the checkbox at the top of thumbnail (it should become black) and then click on the "Segment" button just above.
+
+As explained [in the documentation](segment.md#overview-of-the-segmentation-panel), it will open a panel where we can enter a configuration for the segmentation and layout detection task.
+
+To familiarize with the segmentation interface, we will start with solely detecting the lines on the image. Select the default segmentation model ("default (blla.mlmodel)"), the "Lines Baselines and Masks" and the "Horizontal l2r" options. Make sure the "Override" box is checked and click on "Segment". While the segmentation task in running, a yellow button appears on the thumbnail: it allows you to stop the process if necessary.
```

Comment:
I don't think that is necessary, it can give the impression that if you leave the web page, the job fails, which is not true server-wise. As long as the job is launched, you can leave the page, and the task will still be processed. You'll just won't have the pop-up box (I think?).